terraform/ec2/main.tf

```diff
@@ -21,6 +21,14 @@ module "common" {
   security_group_name = var.aoc_vpc_security_group
 }
 
+module "aoc_msk_cluster" {
+  source = "../data_aoc_msk"
+
+ cluster_version = var.kafka_version
+ testcase = var.testcase
+ dedup_topic = "ec2${var.testing_type}${var.testing_ami}"
```

Comment:
Should we also be using the testing id as part of the dedup topic?

Reply:
we cannot use testid because there is a finite number of topics supported in kafka, which are created on first use, but are not deleted automatically, therefore we have to reuse topics across tests. 
I still need to figure out how to make this work with the different branches.